src/components/MentionSuggestions.js

```diff
@@ -63,25 +63,42 @@ const MentionSuggestions = (props) => {
      * @returns {JSX.Element}
      */
     const renderSuggestionMenuItem = (item) => {
-        const displayedText = _.uniq([item.text, item.alternateText]).join(' - ');
-        const styledTextArray = getStyledTextArray(displayedText, props.prefix);
+        const isIcon = item.text === CONST.AUTO_COMPLETE_SUGGESTER.HERE_TEXT;
```

Comment:
NAB: this seems clearer to me. 

```suggestion
        const avatarSize = item.text === CONST.AUTO_COMPLETE_SUGGESTER.HERE_TEXT ? CONST.AVATAR_SIZE.MENTION_ICON : CONST.AVATAR_SIZE.SMALLER;
```

Reply:
@szebniok thoughts? Last outstanding thing before I merge.